app.py

```diff
@@ -8,6 +8,8 @@
 from fsd_utils.logging import logging
 from openapi.utils import get_bundled_specs
 
+# random comment for deploy
```

Comment:
to remove? :)

Reply:
Yes, silly me! https://github.com/communitiesuk/funding-service-design-assessment-store/pull/198/commits/e059f802c7ba99c4ac415ee25c8fa937bdd893bf thanks